ncm-spma/src/main/pan/components/spma/functions-ips.pan

```diff
@@ -0,0 +1,553 @@
+# ${license-info}                                                                                                                                                                                                                            
```

Comment:
Yes please.

This PR isn't supposed to be even considered for inclusion yet. As it's just a first-shot review for now.